packages/frontend-api/src/Model/Resolver/Payment/PaymentResolver.php

```diff
@@ -41,7 +41,7 @@ public function __construct(PaymentFacade $paymentFacade, Domain $domain)
     public function resolver(string $uuid): Payment
     {
         try {
-            return $this->paymentFacade->getByUuid($uuid);
+            return $this->paymentFacade->getVisibleByUuid($uuid);
```

Comment:
shouldn't be `PaymentCanBeUsedValidator` updated too?

Reply:
Yes, nice catch!